app/Presenters/AssetPresenter.php

```diff
@@ -110,7 +110,7 @@ public static function dataTableLayout()
                 'visible' => false,
                 'formatter' => 'employeeNumFormatter',
             ],[
-                'field' => 'jobtitle',
+                'field' => 'assigned_to.jobtitle',
```

Comment:
I had to double-check changing this line but it renders in the asset index properly.

<img width="836" height="256" alt="Image" src="https://github.com/user-attachments/assets/8488ba6f-43ac-45ba-8182-798312c29bc0" />